lib/config/options/index.ts

```diff
@@ -2107,6 +2107,16 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'maxRequestsPerSecond',
+    description: 'Limit requests rate per host.',
+    type: 'integer',
```

Comment:
🤔 does the validator accept float for this type?

Reply:
good question.. we need that

Reply:
It supports fractional values, despite of being named as `integer`

Reply:
ok, then we should rename this to `number` later

Reply:
this seems to be the only usage of string `integer` for `type`
https://github.com/renovatebot/renovate/blob/2349e5597626dc230394f00ffddf186c19bde7d9/lib/config/validation.ts#L258-L266